client/my-sites/themes/helpers.js

```diff
@@ -31,22 +26,30 @@ function appendActionTracking( option, name ) {
 	} );
 }
 
-export function getAnalyticsData( path, tier, site_id ) {
-	let basePath = sectionify( path );
+export function getAnalyticsData( path, { filter, vertical, tier, site_id } ) {
+	let analyticsPath = '/themes';
 	let analyticsPageTitle = 'Themes';
 
+	if ( vertical ) {
+		analyticsPath += '/:vertical';
+	}
+
 	if ( tier ) {
-		basePath += '/type/:tier';
+		analyticsPath += '/:tier';
```

Comment:
Looking at the routes, tier only has 2 values, `free` and `premium`. Since this is a finite number of things, we may want to not normalize `tier`. cc @yanirs 

For example we may return `/themes/:vertical/free/filter/:filter` and `/themes/:vertical/premium/filter/:filter`